src/injections/baseThunkAction/index.js

```diff
@@ -2,16 +2,18 @@ function baseThunkAction({
   type,
   target,
   service,
-  payload = () => {}
+  payload = () => {},
+  successData = response => response.data,
+  failureData = response => response.problem
```

Comment:
Add tests to support these selectors.
Also I'd add tests for `emptyThunkAction` and `singleCallAction` described in #11

Reply:
Override this comment